src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemModelTests.cs

```diff
@@ -36,7 +37,23 @@ public void SarifWorkItemModel_PopulatesADODescription()
             var workItemModel = new SarifWorkItemModel(sarifLog, context);
             workItemModel.BodyOrDescription.Should().NotBeNullOrEmpty();
             workItemModel.BodyOrDescription.Should().Contain(nameof(TestData.TestToolName));
-            workItemModel.BodyOrDescription.Should().Contain("To see result details, please visit the Scans tab of this bug, or the attached SARIF log.");
+            workItemModel.BodyOrDescription.Should().Contain(sarifLog.Runs[0].VersionControlProvenance[0].RepositoryUri.OriginalString);
+            workItemModel.BodyOrDescription.Should().Contain("Visual Studio SARIF add-in.");
```

Comment:
>"Visual Studio SARIF add-in." [](start = 61, length = 29)

Another reason for this to be in a resource file: If you change it, your test doesn't start failing.